src/github_client_wrapper.ts

```diff
@@ -64,8 +65,9 @@ export class GitHubClientWrapper implements GitHubClient{
     return pullRequestData.base.ref;
   }
 
-  async get_pull_request(pr_number: number): Promise<Octokit.PullsGetResponse> {
-    const getPrResponse = await this.restClient.pulls.get({
+  // TODO: make this strongly typed
+  async get_pull_request(pr_number: number): Promise<any> {
```

Comment:
I had trouble actually finding what the up-to-date type is for this, but it does work as `any`